.github/workflows/macos.yml

```
@@ -71,9 +85,37 @@ jobs:
 
       - run: make
 
+      # Launchable CLI requires Python and Java
+      # https://www.launchableinc.com/docs/resources/cli-reference/
+      - name: Set up Python
+        uses: actions/setup-python@871daa956ca9ea99f3c3e30acb424b7960676734 # v5.0.0
+        with:
+          python-version: "3.10"
+      - name: Set up Java
+        uses: actions/setup-java@7a445ee88d4e23b52c33fdc7601e40278616c7f8 # v4.0.0
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: Setup Launchable
```

```suggestion
      - name: Set up Launchable
```

I noticed the one-word "setup" is used in this step name: "Setup Launchable", while the 2 words "set up" is used for other step names: "Set up Python" and "Set up Java " for other step names. I can see interesting results by doing `grep` by "setup" and "set up". And yes, the test-unit uses `def setup` as verb.

```
$ grep -ri 'setup' .github/workflows/
$ grep -ri 'set up' .github/workflows/
```

The following document says below as a reference.

https://writingexplained.org/setup-vs-set-up-difference 
> Setup (one word) is a [noun](https://writingexplained.org/grammar-dictionary/noun) ...
> ...
> The one-word setup is sometimes hyphenated to appear as set-up. There is nothing wrong with this, but it much more frequently appears as a single word, especially in American English.

> Set up (two words) is a [verb phrase](https://writingexplained.org/grammar-dictionary/phrasal-verb) ...

I think that I or you need to port some small changes that we found in this PR into the the ruby/debug and ruby/vscode-rdbg later after merging this PR to align the style between ruby/* repositories.

---

Updated

---

I'm going to update them in ruby/debug in ruby/vscode-rdbg, too.